packages/sui-studio/src/routes.js

```diff
@@ -12,9 +12,12 @@ import TestPage from './components/test-page/index'
 
 export default (
   <Route>
+    <Redirect
```

Comment:
Sadly, we've removed support for nested Redirects and `IndexRedirect`. This is the way to keep the same functionality.